src/Microsoft.Azure.WebJobs.Logging/ILogTableProvider.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+using Microsoft.WindowsAzure.Storage.Table;
+
+namespace Microsoft.Azure.WebJobs.Logging
+{
+    /// <summary>
+    /// Logs are stored across multiple tables to aide in purging old logs.  
+    /// </summary>
+    public interface ILogTableProvider
+    {
+        /// <summary>
+        /// Return a table with the given suffix. The logging will create this table if it doesn't exist.
+        /// </summary>
+        /// <param name="suffix">Suffix will be legal table name characters. </param>
```

Comment:
"The suffix to use for the log table name. Should only include legal table name characters."